common/frame_manager.go

```diff
@@ -697,10 +697,9 @@ func (m *FrameManager) WaitForFrameNavigation(frame *Frame, opts goja.Value) api
 		"fmid:%d fid:%s furl:%s",
 		m.ID(), frame.ID(), frame.URL())
 
-	rt := k6common.GetRuntime(m.ctx)
 	parsedOpts := NewFrameWaitForNavigationOptions(time.Duration(m.timeoutSettings.timeout()) * time.Second)
 	if err := parsedOpts.Parse(m.ctx, opts); err != nil {
-		k6common.Throw(rt, fmt.Errorf("failed parsing options: %w", err))
+		k6Throw(m.ctx, "wait for frame navigation failed to parse options: %v", err)
```

Comment:
Same here about standardizing the message. Maybe `failed parsing waitForFrameNavigation options`?

Let's use the JS function name since that's the user-facing API. Same goes for the other messages.

Reply:
I agree and it should also be explanatory and easy to find for us, what about:

`waitForFrameNavigation cannot wait for event (EventFrameAddLifecycle): %v`